### DISCUSSION TITLE
🖊️ Make empty lists

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#6001] [Language Ideas]
As I discussed with @Felienne over discord, it would be nice if we could make empty lists or lists with just one value and be able to add to that list with the add command.

**Comment by Felienne** [comment]
I like this idea @MarleenGilsing, I can pick it up soon I think, but what do you think is a good syntax for this?

We can do it like Python, do you think that is reasonable?

```
list = []
```

Also, in what level do you think this makes sense?

**Comment by MarleenGilsing** [comment]
This syntax looks nice and clean, so I think it makes sense to use it. 

My first thought would be level 16, in which the brackets are taught, but that is very late, so lets move it up. We are introducing lists in level 5, but that level is quite full already, so I think level 6 might be best. Level 6 only has + - * / and = now, so it's a bit boring anyway, so an empty list would be a nice addition.
